cmd/gardener-extension-networking-cilium/app/app.go

```diff
@@ -40,6 +44,9 @@ import (
 	"github.com/gardener/gardener-extension-networking-cilium/pkg/healthcheck"
 )
 
+// Name is a constant for the name of this component.
+const Name = "gardener-extension-networking-calico"
```

Comment:
Please correct the name.

```suggestion
const Name = "gardener-extension-networking-cilium"
```